lane.py

```diff
@@ -406,19 +406,18 @@ def __init__(self, library, collection, availability, order,
         a per-library basis.
         """
         super(Facets, self).__init__(entrypoint, entrypoint_is_default)
-        if order_ascending is None:
-            if order == self.ORDER_ADDED_TO_COLLECTION:
-                order_ascending = self.ORDER_DESCENDING
-            else:
-                order_ascending = self.ORDER_ASCENDING
-
         collection = collection or self.default_facet(
             library, self.COLLECTION_FACET_GROUP_NAME
         )
         availability = availability or self.default_facet(
             library, self.AVAILABILITY_FACET_GROUP_NAME
         )
         order = order or self.default_facet(library, self.ORDER_FACET_GROUP_NAME)
+        if order_ascending is None:
+            if order in Facets.ORDER_DESCENDING_BY_DEFAULT:
+                order_ascending = self.ORDER_DESCENDING
+            else:
+                order_ascending = self.ORDER_ASCENDING
```

Comment:
There is currently special code in circulation for changing the order of a crawlable feed. This establishes a general rule that feeds ordered by dates should be ordered "newest first" by default, which lets me get rid of that special code.